scripts/build.sh

```diff
@@ -1,17 +1,23 @@
 #!/usr/bin/env bash
 
-if [ -z "$1" ]; then 
+if [ -z "$1" ]; then
     BUILD_TYPE=Release
-else 
+else
     BUILD_TYPE="$1"
 fi
 
-cd $(dirname "$0")
-mkdir -p build
-cd build
-cmake .. -DCMAKE_BUILD_TYPE="$BUILD_TYPE"
+cd $(dirname "$0")/.. &&
```

Comment:
honestly the better way is to do this the way I originally had this which was to go to the git directory root (aka the project root). `git rev-parse --show-toplevel`